libmscore/slur.cpp

```diff
@@ -320,8 +320,9 @@ void SlurSegment::computeBezier(QPointF p6o)
       QPointF start = pp1;
       int nbShapes  = 32;  // (pp2.x() - pp1.x()) / _spatium;
```

Comment:
Reducing this value here should also give a signifficant performance gain. 32 points for every slur/tie, longg or short, seems way too much

Reply:
This gives a performance gain but not so significant compared to the `pointAtPercent` optimization. Even reducing this value to 4 rectangles gains only about 5-6 additional per cent from the original time compared to the version with 32 shape rectangles (that is, about 33-34% layout time reducing instead of 27-28% but results are not so stable, maybe some more iterations needed to measure this better). It could be good to reduce a number of rectangles, at least based on the slur's size or some given spatial precision of approximation. Still just reducing this number affects a precision of this approximation so I didn't include it in this PR. I believe this question should be considered better separately (but it looks like it really should be considered).

Reply:
If i remember right the problem was the slur start/end which needed higher resolution to be able to place articulations close enough.